lib/reporter/csv.js

```diff
@@ -0,0 +1,36 @@
+const { format } = require("@fast-csv/format");
```

Comment:
We don't need a library for that since we are just piping it to stdout. If we use regular streams it should be fast enough for users.